table/queries/derived_satellite_scans.sql

```diff
@@ -73,6 +73,20 @@ CREATE TEMP FUNCTION AnswersSignature(answers ANY TYPE) AS (
   ), ",")
 );
 
+CREATE TEMP FUNCTION BadResolver(
```

Comment:
Could you explain (perhaps in a comment), what those rates are? What are the numerator and denominator?

We need our evaluation to be understandable easily, so we need to explain these rates.

Reply:
Added a comment